src/main.rs

```diff
@@ -122,6 +124,16 @@ fn run() -> i32 {
         },
     };
 
+    if let Some(delay) = matches.opt_str("d") {
+        match delay.parse::<u64>() {
+            Ok(secs) => thread::sleep(std::time::Duration::from_secs(secs)),
+            Err(_) => {
+                eprintln!("Invalid delay specified");
```

Comment:
It would be nice to print `delay` here: eprintln!("Invalid delay \"{}\" specified", delay);

Reply:
Yeah I didn't do it initially because most errors don't do that. Fixed & branch updated.